snap/snapcraft.yaml

```diff
@@ -13,21 +13,23 @@ apps:
   srsepc:
     command: usr/bin/srsepc
 layout:
-  /usr/share/srslte:
-    bind: $SNAP/usr/share/srslte
-  /etc/srslte:
-    bind: $SNAP/usr/share/srslte
+  /usr/share/srsran:
+    bind: $SNAP/usr/share/srsran
+  /etc/srsran:
+    bind: $SNAP/usr/share/srsran
 parts: 
-  srslte:
+  srsran:
     plugin: cmake
     cmake-parameters:
       - -DCMAKE_BUILD_TYPE=RelWithDebInfo
       - -DENABLE_ASAN=True
       - -DCMAKE_INSTALL_PREFIX=/usr
```

Comment:
Can we remove this parameter and the `prime` section altogether? Sorry, this is not directly related to this PR.

Reply:
On a similar note, does it work with `core22` instead of `core20`?

Reply:
core22 wouldn't be able to find `libzmq3-dev`.
There would be solutions for that, like building zeromq from source :)

Reply:
> Can we remove this parameter and the `prime` section altogether? Sorry, this is not directly related to this PR.

This was added to have the example config files in the `usr` directory, but we could test other options to copy those files :)

Reply:
We can keep it as is, it doesn't matter much honestly